src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/UsePropertiesWhereAppropriateTests.cs

```diff
@@ -341,7 +342,11 @@ public async Task VisualBasic_CA1024NoDiagnosticOnUnboundMethodCaller()
         Return 0
     End Function
 End Class
-", CompilerDiagnostics.None);
+",
+                // Test0.vb(3,21): warning CA1024: Use properties where appropriate
+                GetCA1024BasicResultAt(3, 21, "GetSomethingWithUnboundInvocation"),
+                // Test0.vb(4) : error BC30451: 'Console' is not declared. It may be inaccessible due to its protection level.
```

Comment:
Adding the `Imports System` will prevent the unbound method call.

Reply:
Yes, though this test is supposed to be testing this case.